hi-ml-histopathology/src/histopathology/utils/metrics_utils.py

```diff
@@ -29,7 +29,8 @@ def select_k_tiles(results: Dict, n_tiles: int = 5, n_slides: int = 5, label: in
     :param n_tiles: number of tiles to be selected for each slide
     :param n_slides: number of slides to be selected
     :param label: which label to use to select slides
-    :param select: criteria to be used to sort the slides (select[0]) and the tiles (select[1])
+    :param highest_pred: criteria to be used to sort the slides by highest prediction score
```

Comment:
can you change the naming for the boolean args? "highest_pred" sounds like it is a floating number. "use_highest_pred". And also say in doc what highest_pred==True/False actually means.